src/main/scala/scalismo/mesh/TriangleMesh.scala

```diff
@@ -85,6 +85,8 @@ case class TriangleMesh3D(pointSet: UnstructuredPointsDomain[_3D], triangulation
   val triangles = triangulation.triangles
   val cells = triangles
 
+  lazy val operations = MeshOperations(this)
```

Comment:
It would be good to explicitly state here that the return type is a TriangleMeshOperation (and not a generic MeshOperation)